ui/preferences/src/main/res/layout/choose_home_screen_order_dialog.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="12sp">
```

Comment:
You can apply the padding directly on the RecyclerView, then the layout is more flat